### PR TITLE
Illusionary Form changes

### PR DESCRIPTION
**Pull request by GarOfMeridian**
Here is a shot at fixing up illusionary form and poison a little bit
without completely removing proc effects. The primary concern with this
particular step is getting players to illusionary form into more impressive
art than a silhouette of a rat. Seriously, our combat looks ridiculous
when all our PvP videos are of little rat silhouettes clawing the air
at each other. This will be much more impressive.

* Dusk rats can no longer be morphed or illusionary formed.

* There is now a new type of enemy called a corrupted avar chieftain.
It's a stronger avar chieftain with a poison equal to the dusk rat's.
It can be morphed into with the word teotkoavar. The corrupted avar
chieftain cannot multi-poison a player (only one poison at a time).
These will spawn at Ruins of the Riija temple and other avars will fight
them if prompted. Corrupted avar chieftains drop mainly cyan mushrooms,
nerudite arrows, and dark angel feathers, thus being the only source
on the island for that reagent. They are also -95 karma, so they will
take players to +95, being a rare "shal killable" mob on the island.

With this first step, hopefully players will be in cooler art, and the 
one-poison limit should significantly dampen the "force multiplier"
effect of using this with reflections.

There is also a negative here: avar chieftains are huge. Players
will stand out rather than being hard to see like dusk rats.

If this isn't enough, further changes might have to follow.

Tested locally

### CODE REVIEW

File: kod/object/active/holder/nomoveon/battler/monster/avchiefpoison.kod [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+      return random(8,15);
+   }
+
+   HitSideEffect(what=$)
+   {
+      local oSpell;
+
+      if Random(1,POISON_CHANCE) = 1
+         AND IsClass(what,&Player)
+         AND NOT Send(what,@IsEnchanted,#byClass=&Poison);
```
Comment (skittles1) [inline kod/object/active/holder/nomoveon/battler/monster/avchiefpoison.kod]: Need to remove the ; here (doesn't compile).

**Comment by skittles1** [comment]
Also, this has merge conflicts with #818 and #832.

**Comment by GarOfMeridian** [comment]
; removed, but no idea what to do about merge conflicts

**Comment by skittles1** [comment]
Come in IRC, organise with keen and diggie where to move things I guess... Morbus already removed some changes that were conflicting with the shal/qorbane move.

**Comment by GarOfMeridian** [comment]
seems easier to just skip this one for a moment and i'll rebase after their stuff is in

**Comment by skittles1** [comment]
Yeah that's pretty much what I had in mind, put in the content we have in (at least) two parts and verify carefully we aren't overwriting/removing anything.

On a related note it'd be really good if you could update your master branch and rebase your open pull requests/respond to line note comments so we can see which ones are compatible with 1.0.8.0.

**Comment by GarOfMeridian** [comment]
ok

i need to do some more lore work on this pull, too re: http://openmeridian.org/forums/index.php/topic,406.0.html

**Comment by GarOfMeridian** [comment]
rebased

**Pull request closed without merging**